surefire-provider/pom.xml

```diff
@@ -27,6 +27,7 @@
     <dependency>
       <groupId>org.arquillian.smart.testing</groupId>
       <artifactId>smart-testing-core</artifactId>
+      <version>${project.version}</version>
     </dependency>
     <dependency>
       <groupId>org.arquillian.smart.testing</groupId>
```

Comment:
I think that the following strategy dependencies are not necessary now - as it should be completely isolated. 
They should be moved to the functional tests - to have them on classpath. It would be good to have some depchain not to have 4 dependencies but only one...